core/crypto/crypto_test.go

```diff
@@ -304,20 +313,144 @@ func TestClientGetAttributesFromTCert(t *testing.T) {
 		t.Fatalf("Cert should have length > 0")
 	}
 
-	attributeBytes, err := deployer.ReadAttribute("company", tcertDER)
+	attributeBytes, err := abac.GetValueForAttribute("company", tcert.GetPreK0(), tcert.GetCertificate())
```

Comment:
Shall we avoid to expose PreK0 that way? May CertificateHandler have a method GetValueForAttribute(attrName) directly?